gdk/commands/component/PublishCommand.py

```diff
@@ -67,6 +67,8 @@ def upload_artifacts_s3(self, component_name, component_version):
         try:
             bucket = self.project_config["bucket"]
             region = self.project_config["region"]
+            s3_upload_file_args = self.project_config.get("s3_upload_file_args")
```

Comment:
I just have a comment on the name chosen for the configuration. Currently on the config names there is no reference to `s3`. For instance we do no call `bucket` `s3_bucket`, why start using it for this one?

Reply:
why not call it `args`? if publish is already in the context of s3

Reply:
This configuration here is very specific to s3 client `upload_file` operation. `args` is too generic in that sense. We want to make it as clear as possible that the args here will be used only during s3 file upload. "publish" is more in the context of creating a new component version where s3 file upload is a sub-task.

Reply:
Added `file_upload_args` under `options` as discussed offline.